implementations/rust/ockam/ockam_entity/src/profile_state.rs

```diff
@@ -21,6 +33,8 @@ pub struct ProfileState {
     contacts: Contacts,
     vault: VaultSync,
     key_attribs: KeyAttributes,
+    signing_key: BbsSecretKey,
```

Comment:
This is what I'm trying to get_rid in my PR. Should we merge this first and then mine, or just remove this temporary so I could revive it properly?